.github/workflows/e2e.yml

```diff
@@ -0,0 +1,23 @@
+name: End-to-end tests
+on: [push]
+jobs:
+  cypress-run:
+    runs-on: ubuntu-20.04
+    env:
+      POSTGRES_URL: "foobar"
+      COOKIE_SECRET: "foobar"
+      MAILER_HOST: "foobar"
+      MAILER_PORT: "321"
+    # stop the job if it runs over 10 minutes to prevent a hanging process from using all your CI minutes
+    timeout-minutes: 10
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      # Install NPM dependencies, cache them correctly
+      # and run all Cypress tests
+      - name: Cypress run
+        uses: cypress-io/github-action@v2
+        with:
+          build: npm run build
+          start: npm run start
```

Comment:
don't these need to be `front-end:build` and `front-end:watch` ? Are they working as is?

Reply:
I added a convenience script to the `package.json` so `build` runs both the front- and back-end builds.

Re `watch`, I ran `start` instead because I figured we don't need to watch for front-end changes on CI. I also thought I needed to start both the back-end too, but maybe I don't?

Reply:
They do appear to be working on CI, yes

Reply:
Was my mistake I hadn't got to the `package.json` yet to see the new scripts

Reply:
Re running the back-end, I've looked through the front-end scripts in the package.json and I don't think it's possible to run the front-end without the back-end. front-end:watch just watches the files for changes, and if you run it on its own, you can't get to localhost:3000.